### ISSUE TITLE
PURI compatibility question.

### ISSUE THREAD

**Issue by hyotang666** [fukamachi/quri#56]
merge-uris set path slot with an empty string (i.e. "") in the cases below.

(asdf:system-version (asdf:find-system :quri)) => "0.4.0"

(quri:uri-path (quri:merge-uris "https://google.com" "")) => ""
(quri:uri-path (quri:merge-uris "https://google.com" "/")) => ""
(quri:uri-path (quri:merge-uris "https://google.com" ".")) => ""

PURI always set it `NIL`.

(puri:uri-path (puri:merge-uris "https://google.com" "")) => NIL
(puri:uri-path (puri:merge-uris "https://google.com" "/")) => NIL
(puri:uri-path (puri:merge-uris "https://google.com" ".")) => NIL

If 'replacement of PURI' in the README means 'compatible for PURI', merge-uris should set path with NIL.

Is this expected behavior?

BTW, FYI, I found dex:get fails with an empty string path.

(dex:get (quri:merge-uris "https://google.com" "")) => 404 bad request.
Works fine with NIL or "/" path slot.

So, in other words, which one should respond to it?

**Comment by fukamachi** [comment]
> If 'replacement of PURI' in the README means 'compatible for PURI',

No, QURI is written as a replacement for PURI, but it does not promise that QURI is compatible with PURI since PURI is dead now and QURI has to evolve.

The problem of communication errors with Dexador for URIs with undefined `uri-scheme` should be handled in Dexador.

**Comment by hyotang666** [comment]
Ok.
Thank you!
